modules/atomic-widgets/widgets/atomic-heading.php

```diff
@@ -67,13 +67,13 @@ public function get_atomic_controls(): array {
 
 		$title_control = Textarea_Control::bind_to( 'title' )
 			->set_label( __( 'Title', 'elementor' ) )
-			->set_placeholder( __( 'Type your title here', 'elementor' ) );
+			->set_placeholder( __( 'Write something or press ‘space’ for AI,  ‘/’ for Dynamic content', 'elementor' ) );
```

Comment:
```suggestion
			->set_placeholder( __( "Write something or press 'space' for AI,  '/' for Dynamic content", 'elementor' ) );
```

easier for translators